playground/current_user.gd

```diff
@@ -1,30 +1,30 @@
 extends Panel
 
-@onready var discord : DiscordSDK = get_node("/root/Discord")
+@onready var discord: DiscordSDK = get_node("/root/Discord")
 @onready var avatar: TextureRect = $Avatar
 @onready var username: RichTextLabel = $Name
 @onready var flags: RichTextLabel = $Flags
 
 
 func _ready() -> void:
-	discord.connect("dispatch_current_user_update", Callable(self, "update"))
+	discord.dispatch_current_user_update.connect(update)
 
 
-func update(data) -> void:
+func update(data: DiscordSDK.CurrentUserUpdateData) -> void:
 	username.text = data["global_name"] + "  [i](@" + data["username"] + ")[/i]"
 	flags.text = data["id"]
```

Comment:
Should use property access instead of indexing here

```suggestion
	username.text = data.global_name + "  [i](@" + data.username + ")[/i]"
	flags.text = data.id
```